app/js/rest/patientHeaderRest.js

```diff
@@ -0,0 +1,23 @@
+import axiosInstance from '../config/axios';
+
+const api = {
+  getPatientRecord: params => axiosInstance.get(`patient/${params.patientUuid}?v=custom:(patientId,uuid,patientIdentifier:(uuid,identifier),person:(gender,age,birthdate,birthdateEstimated,personName,preferredAddress),attributes:(value,attributeType:(name)))`)
```

Comment:
Let's rename this getPatient and actually move into patientRest.js in react-components:

https://github.com/openmrs/openmrs-react-components/blob/master/src/rest/patientRest.js